platform/core-api/src/com/intellij/psi/CommonClassNames.java

```diff
@@ -132,5 +132,10 @@ public interface CommonClassNames {
 
   String CLASS_FILE_EXTENSION = ".class";
 
+  String JAVA_TIME_LOCAL_DATE = "java.time.LocalDate";
```

Comment:
We use `CommonClassNames` only for class names that are actually common. These names are not (so far at least). Could you remove them from this class?

Reply:
Deleted in `CommonClassNames`